docs/changelog/CHANGELOG.md

```diff
@@ -32,6 +32,7 @@ See also [LTS releases](https://docs.victoriametrics.com/lts-releases/).
 * BUGFIX: [vmgateway](https://docs.victoriametrics.com/vmgateway/): fix possible panic during parsing of a token without `vm_access` claim. This issue was introduced in v1.104.0.
 * BUGFIX: [vmui](https://docs.victoriametrics.com/#vmui): fix error messages rendering from overflowing the screen with long messages. See [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/7207).
 * BUGFIX: [vmctl](https://docs.victoriametrics.com/vmctl/): properly add metrics tags for `opentsdb` migration source. Previously it could have empty values. See [this PR](https://github.com/VictoriaMetrics/VictoriaMetrics/pull/7161).
+* BUGFIX: [vmagent](https://docs.victoriametrics.com/vmagent/): fixed hot-reload on `max_scrape_size` change. See [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/7260).
```

Comment:
```suggestion
* BUGFIX: [Single-node VictoriaMetrics](https://docs.victoriametrics.com/)  and [vmagent](https://docs.victoriametrics.com/vmagent/): properly update `max_scrape_size` param change during [hot-reload](https://docs.victoriametrics.com/vmagent/#configuration-update). See [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/7260).
```

Reply:
@AndrewChubatiuk @f41gh7 please pay more attention to the changelog lines. Their main purpose is to be clear for readers. I've updated https://docs.victoriametrics.com/contributing/#pull-request-checklist to highlight this requirement:
> A line in the [changelog](https://docs.victoriametrics.com/changelog/#tip) mentioning the change and related issue in a way that would be clear to other readers even if they don't have the full context.